package.json

```diff
@@ -11,8 +11,8 @@
     "build": "npm run clean && npm run build:commonjs",
     "bundle": "mkdir -p dist && npm run build:umd && npm run build:umd:min",
     "lint": "eslint src",
-    "preversion": "npm run lint && npm test && npm run bundle && git add dist/ && git commit -m 'Publish: build bower distribution'",
-    "prepublish": "npm run build",
+    "preversion": "npm run lint && npm test && npm run build && npm run bundle && git add dist/ && git commit -m 'Publish: build bower distribution'",
+    "prepublish": "npm run build && npm run bundle",
```

Comment:
running bundle in `prepublish` looks redundant to me, because it is anyway run in `preversion`. Or do I miss something?